langchain4j-core/src/main/java/dev/langchain4j/store/embedding/IngestionResult.java

```diff
@@ -0,0 +1,24 @@
+package dev.langchain4j.store.embedding;
+
+import dev.langchain4j.model.output.TokenUsage;
+
+import static dev.langchain4j.internal.ValidationUtils.ensureNotNull;
+
+/**
+ * Represents the result of a {@link EmbeddingStoreIngestor} ingestion process.
+ */
+public class IngestionResult {
+    /**
+     * The token usage information.
+     */
+    private final TokenUsage tokenUsage;
+
+
+    public IngestionResult(TokenUsage tokenUsage) {
+        this.tokenUsage = ensureNotNull(tokenUsage, "tokenUsage");
```

Comment:
```suggestion
        this.tokenUsage = tokenUsage;
```

There are `EmbeddingModel` implementations that do not return `TokenUsage` (e.g. `OllamaEmbeddingModel`). This will fail then.

Reply:
makes sense! done! thank you so much! :)